python_ta/transforms/type_inference_visitor.py

```diff
@@ -230,9 +230,10 @@ def _handle_call(self, node, func_name, *args):
         try:
             func_type = self.type_store.lookup_function(func_call, *arg_types)
         except KeyError:
-            return TypeInfo(
-                TypeErrorInfo(f'Function {func_call} not found with given args:\
```

Comment:
This part shouldn't be removed altogether (useful for generic function calls). Instead, use a dictionary (see comment below) and only emit the *new* error message when the operator name is in the dictionary; otherwise, use the original error message.